Singer.API/Services/MockUserService.cs

```diff
@@ -0,0 +1,176 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Singer.DTOs;
+using Singer.Helpers;
+using Singer.Helpers.Extensions;
+using Singer.Models;
+using Singer.Services.Interfaces;
+
+namespace Singer.Services
+{
+   public class MockUserService : IUserService
+   {
+      #region FIELDS
+
+      private readonly IList<CareUser> _mockData = new List<CareUser>
+      {
+         //new CareUser
+         //{
+         //   Id = Guid.NewGuid().ToString(),
+         //   Name = "Joske Vermeulen",
+         //   BirthDay = DateTime.Parse("06/07/2008", CultureInfo.InvariantCulture),
+         //   CaseNumber = "0123456789",
+         //   AgeGroup = Models.AgeGroup.Child,
+         //   IsExtern = false,
+         //   HasTrajectory = true,
+         //   HasNormalDayCare = true,
+         //   HasVacationDayCare = true,
+         //   HasResources = true
+         //},
+         //new CareUser
+         //{
+         //   Id = Guid.NewGuid().ToString(),
+         //   Name = "Kim Janssens",
+         //   BirthDay = DateTime.Parse("08/07/2006", CultureInfo.InvariantCulture),
+         //   CaseNumber = "9876543210",
+         //   AgeGroup = Models.AgeGroup.Child,
+         //   IsExtern = true,
+         //   HasTrajectory = true,
+         //   HasNormalDayCare = true,
+         //   HasVacationDayCare = true,
+         //   HasResources = true
+         //},
+         //new CareUser
+         //{
+         //   Id = Guid.NewGuid().ToString(),
+         //   Name = "Benjamin Vermeulen",
+         //   BirthDay = DateTime.Parse("06/08/2010", CultureInfo.InvariantCulture),
+         //   CaseNumber = "091837465",
+         //   AgeGroup = Models.AgeGroup.Youngster,
+         //   IsExtern = false,
+         //   HasTrajectory = true,
+         //   HasNormalDayCare = true,
+         //   HasVacationDayCare = true,
+         //   HasResources = false
+         //},
+      };
+
+      #endregion FIELDS
+
+
+      #region CONSTRUCTOR
+
+      #endregion CONSTRUCTOR
+
+
+      #region METHODS
+
+      public async Task<T> CreateUserAsync<T>(T createUser) where T : CareUser
+      {
+         // TODO User UserManager
```

Comment:
Should this TODO be solved before merging?

Reply:
Yes and no. I took the easy option in this case, and just let the implementation happen when the actual ticket is implemented.